infra/src/main/resources/db/migration/prod/V129__extend_operational_point_table.sql

```diff
@@ -102,13 +103,13 @@ insert into layout.operational_point_id (
 
 insert into layout.operational_point
   (id, draft, design_id, layout_context_id, name, abbreviation, uic_code, type, location,
-   state, rinf_type_code, polygon, origin)
+   state, rinf_type, polygon, origin)
   (
     select
       row_number() over (order by external_id),
-      false,
+      true,
       null,
-      'main_official',
+      'main_draft',
```

Comment:
Tässä Jyrkin mielipiteen mukaisesti että alkutilassa kaikki 560 pistettä luonnoksina. Periaatteessahan tätä ei välttämättä tarvitsisi tehdä migrassa näin lainkaan, kun kyllähän Ratkosta tulevat pisteet päätyvät muutenkin layout-tauluun kunhan integraatio ensimmäisen kerran ajetaan, mutta alkumigra kannattaa kyllä tehdä sekvenssissä siksi, että sitten pystyy migroimaan sekvenssissä myös muista lähteistä haetut alkutiedot.

Tungin muutoksen tähän pullariin siis siksi, ettei tarvitse useampaa kertaa käydä deviä puskemassa takaisin pystyyn migramuutoksista.